modules/microshift-low-latency-concept.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-low-latency-concept_{context}"]
+= Lowering latency in {microshift-short} applications
+
+Latency is defined as the time from an event to the response to that event. You can use low latency configurations and tuning in a {microshift-short} cluster running in an operational or software-defined control system where an edge device has to respond quickly to an external event. You can fully optimize low latency performance by combining {microshift-short} configurations with operating system tuning and workload partitioning.
+
+[IMPORTANT]
+====
+The CPU set for management applications, such as the {microshift-short} service, OVS, CRI-O, {microshift-short} pods, and isolated cores, must contain all-online CPUs.
+====
+
+[id="microshift-low-latency-workflow_{context}"]
+== Workflow for configuring low latency for {microshift-short} applications
+To configure low latency for applications running in a {microshift-short} cluster, you must complete the following tasks:
+
+Required::
+* Install the `microshift-low-latency` RPM.
+* Configure workload partitioning.
+* Configure the `kubelet` section of the `config.yaml` file in the `/etc/microshift/` directory.
+* Configure and activate a TuneD profile. TuneD is a {op-system-base-full} service that monitors the host system and optimizes performance under certain workloads.
+* Restart the host.
```

Comment:
Can we include links here to each of these procedures to make sure folks go to the right places?

Reply:
Not in a module, I can't (no xrefs). But these are in order throughout. If we get the "latest" functionality on the new website, I'll come back with links.

Reply:
D'oh. How did I miss that?!?